core/src/components/searchbar/searchbar.tsx

```diff
@@ -677,7 +711,7 @@ export class Searchbar implements ComponentInterface {
         class="searchbar-cancel-button"
       >
         <div aria-hidden="true">
-          {theme === 'md' ? (
+          {theme === 'md' || theme === 'ionic' ? (
```

Comment:
Searchbar is currently configured by theme, but the Figma design shows it by platform. We will likely want to update this when the design is finished to change out the icon/buttons when `mode` is `ios` or `md`.